src/utils/externalLinks.js

```diff
@@ -1,21 +1,16 @@
-import i18next from 'i18next';
 import history from '../history';
 import routesReg from './routes';
-import { errorToastDisplayed } from '../actions/toaster';
-import store from '../store';
 
 export default {
   init: () => {
     const { ipc } = window;
 
     if (ipc) {
       ipc.on('openUrl', (action, url) => {
-        const normalizedUrl = url.toLowerCase().replace('lisk://', '/');
```

Comment:
If you click on `Lisk://main/transactions` this won't find the route. any IM may correct the URL from `lisk://` to `Lisk://`